volmigrate/schema_migrations.go

```diff
@@ -0,0 +1,62 @@
+package volmigrate
+
+import (
+	"sort"
+	"sync"
+
+	"github.com/contiv/volplugin/volmigrate/backend"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+var versionCheckLock sync.Mutex
+var usedVersionNumbers map[int64]bool
+
+func init() {
+	usedVersionNumbers = make(map[int64]bool)
+	registerMigrations()
+	sort.Sort(Migrations(availableMigrations))
+}
+
+// registerMigration makes volmigrate aware of a new migration that can be run.
+// If the version number has already been used, it will log a fatal error.
+func registerMigration(version int64, description string, f func(backend backend.Backend) error) {
```

Comment:
uint would still allow 0, so I'd still need to check anyways.  0 is the schema version of an etcd keyspace which has never run any migrations.